### PR TITLE
refactor: replace chakra packages with MUI, convert home page

### PR DESCRIPTION
**Pull request by ryanntannn**
## Description

- Remove chakra packages
- Add MUI packages
- Convert home page to MUI

### CODE REVIEW

**Comment by ryanntannn** [comment]
ignore the build failure, unmigrated pages still have chakra imports and this PR does not merge into develop

File: features/navigation/index.tsx [hunk 1]
```diff
@@ -1,41 +1,94 @@
...
+      <Button onClick={() => router.push("/")}>Home</Button>
+      <Button onClick={() => router.push("/events")}>Events</Button>
+      <Button onClick={() => router.push("/blog")}>Blog</Button>
+      <Button
+        onClick={() =>
+          router.push("https://form.gov.sg/61ea66776c89fd001206bed0")
+        }
+      >
+        Join us
+      </Button>
```
Comment (natashaannn) [inline features/navigation/index.tsx:37]: Small suggestion but the button components seem quite repeated - can consider just mapping across the relevant info. Might also be good to standardize the width of the button in this manner cos right now while the button spacing is consistent, the inconsistency of the button sizing makes the justification a little strange:

image

File: features/navigation/index.tsx [hunk 2]
```diff
@@ -1,41 +1,94 @@
...
+        color="warning"
+      >
+        InnoFest 22/23
+      </Button>
+      {children}
+    </Stack>
+  );
+}
+
+function MobileMenu() {
```
Comment (natashaannn) [inline features/navigation/index.tsx:50]: This is really nice haha thanks for being sensitive to mobile responsiveness!

**Review by natashaannn — APPROVED** [review]
Nice job!! I really like the new nav bar and also the mobile variation!

File: features/navigation/index.tsx (hunk 1 above)
Comment (ryanntannn) [reply features/navigation/index.tsx:37]: will make the changes before merging!

**Pull request merged**